### ISSUE TITLE
Issue about the instance generation process

### ISSUE THREAD

**Issue by YunhaoZhang-Mars** [DAMO-DI-ML/KDD2023-DCdetector#15]
![屏幕截图 2023-09-17 175121](https://github.com/DAMO-DI-ML/KDD2023-DCdetector/assets/44700045/e40aa69a-d9dd-4b38-9806-139ce3eea42f)
Thanks for providing the source code of the great work.
I have a question about instance generation: it seems that the parameter 'step' is never used. So the instances on all datasets were generated by a sliding window with step 1. Am I right?

**Comment by tianzhou2011** [comment]
That's correct. Unfortunately, we lack sufficient computational resources to perform a grid search for the step size. However, it seems that using a step size of 1 already provides us with a reasonable result.

**Comment by yyysjz1997** [comment]
> ![屏幕截图 2023-09-17 175121](https://user-images.githubusercontent.com/44700045/268494653-e40aa69a-d9dd-4b38-9806-139ce3eea42f.png) Thanks for providing the source code of the great work. I have a question about instance generation: it seems that the parameter 'step' is never used. So the instances on all datasets were generated by a sliding window with step 1. Am I right?

Thanks. In the default configuration, the step size is set to 1 for all datasets except the SMD dataset. The Step parameter is adjusted primarily to enhance the preprocessing's generalization capabilities. If your computing resources are limited, you have the option to fine-tune the step size to expedite model computations, but it may come at the cost of a reduction in model performance.
